config/common.php

```diff
@@ -49,6 +52,23 @@
     ],
     MailerInterface::class => new MailerFactory(),
 
+
+    // Cycle DBAL
+    DatabaseManager::class => new CycleDbalFactory(),
     // Cycle ORM
     ORMInterface::class => new CycleOrmFactory(),
+
+    // Entity dir
+    # todo: replace to class object
```

Comment:
Do you mean a factory?

Reply:
I'm not sure. Maybe here needed make a repository. Like EntityFileRepository that will contain paths to all Entities